build-tools/_build-lib.sh

```diff
@@ -93,7 +93,7 @@ ginkgo_test_with_profile () {
   (
     export GOBIN="$BUILDDIR/bin"
     echodo cd "$WKDIR/src/$PKGIMPORT"
-    echodo ginkgo -r -keepGoing -trace -randomizeAllSpecs -progress --nodes 4 \
+    echodo ginkgo -r -compilers 1 -keepGoing -trace -randomizeAllSpecs -progress --nodes 4 \
```

Comment:
remove the compilers flag, this isn't an issue for us currently and we need more investigation as to whether we need to limit this.

Reply:
done